src/app/passes/pass-list/pass-table-row/pass-table-row.component.html

```diff
@@ -4,9 +4,18 @@
 <td data-label="Email"
     scope="row"
     class="col-3">{{validate('email')}}</td>
+
+<td *ngIf="rowData.facilityType === 'Trail'"
+    data-label="Guests"
+    scope="row"
+    class="col-1">{{validate('numberOfGuests')}}</td>
+<td *ngIf="rowData.facilityType === 'Parking'"
+    data-label="Guests"
+    scope="row"
+    class="col-1">{{validate('license')}}</td>
```

Comment:
Weird issue - if the type of Facility ever changes between Trail/Parking, there will be a combination of licenses and number of guests in the same column but under only one data-label (either Guests or License, depending on current Facility type). Highly doubt this will happen in practice and probably doesn't show up in any passes in `test`. Current .csv export does not have this issue.

Reply:
We addressed this in meeting on the 24th. It's the reason why on edit, we have disabled the ability to edit facility type.